src/components/Footer.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TypeOfLink } from '../App';
+
+type Props = {
+  todosCounter: number;
+  selectedLink: TypeOfLink;
+  setSelectedLink: (arg: TypeOfLink) => void;
+  todos: Todo[];
+  setAllTodos: (arg: Todo[]) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todosCounter,
+  selectedLink,
+  setSelectedLink,
+  todos,
+  setAllTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todosCounter} items left`}
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: selectedLink === TypeOfLink.All,
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setSelectedLink(TypeOfLink.All)}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: selectedLink === TypeOfLink.active,
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => setSelectedLink(TypeOfLink.active)}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: selectedLink === TypeOfLink.completed,
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => setSelectedLink(TypeOfLink.completed)}
+        >
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method